instrumentation/runtime-metrics/library/src/test/java/io/opentelemetry/instrumentation/runtimemetrics/MemoryPoolsTest.java

```diff
@@ -50,6 +59,125 @@ void setup() {
     beans = Arrays.asList(heapPoolBean, nonHeapPoolBean);
   }
 
+  @Test
+  void registerObservers() {
```

Comment:
Also adding some test coverage to make this consistent with the other runtime observers.